MovieApp/MovieApp/ViewControllers/HomePageViewController+Design.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+import UIKit
+import SnapKit
+
+extension HomePageViewController: DesignProtocol {
+
+    func buildViews() {
+        createViews()
+        styleViews()
+        defineLayoutForViews()
+    }
+    
+    func createViews() {
+        headerImageView = UIImageView()
+        
+        flowLayout = UICollectionViewFlowLayout()
+        
+        presenter = HomePagePresenter()
```

Comment:
Presenter, if not injected, should be initialized in the ViewController `init` or as a stored property, not in the `createViews`.